src/renderer/ModalsLayer.js

```diff
@@ -44,7 +44,7 @@ const ModalsLayer = ({ visibleModals }: *) => (
       <BackDrop state={state}>
         {/* {// Will only render at the end of december
           isSnowTime() ? <Snow numFlakes={200} /> : null} */}
-        {visibleModals.map(({ name, data }, i) => {
+        {visibleModals.map(({ name, ...data }, i) => {
```

Comment:
what is the impact of this change on existing stuff? if you can pinpoint one example of thing that got fix with that to make sure QA covers it

Reply:
this should not impact the existing stuff since the props on our current modals aren't accessed at the root of each modal.
This is used however in the new delegation root once we use it on both tezos and :deciduous_tree: